esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -574,20 +574,14 @@ diagnostics:
     description: WOA check
     variables:
       so:
-        mip: Omon
       thetao:
-        mip: Omon
       no3:
-        mip: Oyr
       o2:
-        mip: Oyr
       po4:
-        mip: Oyr
       si:
-        mip: Oyr
     additional_datasets:
-      - {dataset: WOA, project: OBS, tier: 2,
-         type: clim, version: 2013v2, start_year: 2000, end_year: 2000}
+      - {dataset: WOA, project: OBS6, tier: 2, mip: Omon,
```

Comment:
Since the list of cmorized variables has changed, this would need to be changed accordlingly I guess

Reply:
This was updated in 5a9cd64.